### DISCUSSION TITLE
Entity types for PoC workflows

### DISCUSSION THREAD

**Discussion by foundrytom** [OpenAssetIO/OpenAssetIO#609] [Ideas and change proposals]
A gathering point for top 5 entity types we could use to attempt to model Traits/Specifications as part of PoC workflows.

**Comment by antirotor** [comment]
my tips:

1) model - simplest of them all, asset that has multiple representations. You can have a model as fbx/abc/obj. No additional metadata needed.
2) image sequence  - frame based list of files. Needs frame start/frame end/step/start handles/end handles/frame rate. Might include flags about handling non-existent frames in sequence, colorspaces. Might be tricky because you need to distinguish between single frame  used as sequence, single frame using as single image, multiple independent images and image sequence 🙄 
3) geometry cache -  time based alembic file with arbitrary hierarchy inside but with start/end/handles
4) something with sidecar file, like yeti cache (frame based sequence of cache files with one .fur file with metadata for example)
5) something layered - 3d layout. Metadata referencing to other assets to be loaded along with their versions, positions, other relations

**Reply by afrocircus** [reply]
A USD scene file containing on or more usd assets as references might be a good one to include in the POC as well.

**Comment by foundrytom** [comment]
https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation/pull/21 adds a sketch of image sequence proxy discovery via `getRelatedReferences`, this required a bunch of traits to describe these relationships, as well as an image sequence type.  In that PR, there are some vignettes of what these traits/specifications might look like. They're probably not fit for purpose in terms of real production scenarios, but hopefully they illustrate a possible approach for further discussion. Be great to see what you folks think.
